resources/texts/change_log.txt

```diff
@@ -1,3 +1,97 @@
+- Introduced the machine name in the gcode headers, contributed by @smartin015
+
+*Bug Fixes that improve Printed Part Quality
+- The first support layers were printed incorrectly if adhesion was set to None
+- The support was printed before the brim when the origin was at the center of the buildplate
+- Printers with a high resolution would incorrectly print embossed features
+- The flow would unexpectedly increase after a bridge was completed.
+- Bridging settings would not be applied to the first skin layer if the infill density was set to 0
+- The skirt height could collide with some models and could be printed in support
+- A brim would be too small if the extruder was not defined.
+- The Initial Buildplate and Printing Temperature would not be applied correctly when printing One-At-A-Time
+
+*Other Bug Fixes
+- You could not load some Marketplace materials with intents on the like BASFE 17-4PH
+- For some Linux versions it was not possible to add a 3D printer
+- Fixed the installation screen for DMG installation because it still had the old logo.
+- The minimum support area was not working correctly for tree support
+- Support Horizontal Expansion would be hidden but influenced the warning  for Support Interface
+- The shadow in One At A Time printing sequence would not correctly resize with the skirt/brim size
+- It was not possible to select the support structure with basic setting visibility.
+- Removed the option to change the Brim Distance in the per object setting untilit is fixed
+- Fixed a slicing crash if the skirt was larger than the buildplate
+- Fixed a crash that would be caused when rotating a model only a little
+- If support interface is disabled, you can no longer change those settings
+- Ints would be truncated instead of rounded in the engine, contributed by @onitake
+- Fixed a non-raw RegEx pattern string removing a depreciation warning contributed by @cgobat
+
+* Printer definitions, profiles and materials: 
+- For MacOS users these printers are supported again: Elegoo, Strateo3D, Uni, ZAV
+- Added Anycubic Kobra Plus, contributed by @Jordonbc
+- Added Creality Ender-5 S1, contributed  @thomaspleasance
+- Added Entina Tina 2, contributed by @protosam
+- Added Pulse XE E444M, contributed by @randyzwitch
+- Updated Primetower settings for Sovol 2
+- Updated Kingroon KP3S Pro, contributed by @Tachyonn
+- Updated All Goofoo 3D printers to have more nozzles, contributed by @goofoo3d
+- Updated Tree Support settings for Elegoo Printers, Contributed by @ThomasRahm
+- Updated Voron Trident 250, 300 & 350 Voron to include new nozzles, contributed by @zadi15
+- Updated Creality Ender 3 start gcode to prevent bed scratching, Contributed by @PresentMonkey
+- Updated nozzle options for Dagoma Pro 430, contributed by @0r31
+- Updated start gcode and homing behavior in Creality Ender 3 S1, contributed by @GregValiant.
+- Updated faulty disallowed areas for Anycubic Kossel, contributed by @GregValiant
```

Comment:
```suggestion
- Updated Kingroon KP3S Pro, contributed by <a href="https://github.com/Tachyonn">Tachyonn</a>
- Updated All Goofoo 3D printers to have more nozzles, contributed by <a href="https://github.com/goofoo3d">goofoo3d</a>
- Updated Tree Support settings for Elegoo Printers, Contributed by <a href="https://github.com/ThomasRahm">ThomasRahm</a>
- Updated Voron Trident 250, 300 & 350 Voron to include new nozzles, contributed by <a href="https://github.com/zadi15">zadi15</a>
- Updated Creality Ender 3 start gcode to prevent bed scratching, Contributed by <a href="https://github.com/PresentMonkey">PresentMonkey</a>
- Updated nozzle options for Dagoma Pro 430, contributed by <a href="https://github.com/0r31">0r31</a>
- Updated start gcode and homing behavior in Creality Ender 3 S1, contributed by <a href="https://github.com/GregValiant">GregValiant</a>
- Updated faulty disallowed areas for Anycubic Kossel, contributed by <a href="https://github.com/Jordonbc">GregValiant</a>
```